### PR TITLE
 bob.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Question: I received this error in pylint but I'm not sure why: 
"Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)" 

I believe I have a return statement for each conditional. Thoughts?

### CODE REVIEW

File: bob.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    'shout': 'Whoa, chill out!',
+    'shout_question': "Calm down, I know what I'm doing!",
+    'say_nothing': 'Fine. Be that way!',
+    'default': 'Whatever.',
+    }
+
+def hey(phrase):
+    """Return Bob's responses"""
+
+    normalized_phrase = phrase.strip()
```
Comment (amymhaddad) [inline bob.py]: @paulghaddad I noticed that `normalized_phrase = phrase.strip()` works like I have it in the file, but I get an error if I specify that I want to remove whitespaces (ie,` normalized_phrase = phrase.strip(' ')`). Is this b/c the tests include more than just single whitespaces (ie, it includes multiple spaces and tabs in some tests)?

File: bob.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    silence = normalized_phrase == ""
```
Comment (paulghaddad) [inline bob.py]: The logic in lines 16-18 can all be extract into separate, well-named functions!

**Comment by amymhaddad** [comment]
@paulghaddad This exercise is updated.

File: bob.py (hunk 1 above)
Comment (paulghaddad) [reply bob.py]: @amymhaddad Yes, that's absolutely correct.

**Pull request merged**